ta_storage/tests/test_bq.py

```diff
@@ -1,485 +1,381 @@
+        "test-dataset", "test-table", ta_testrun_pb2, ANY
+    )
 
-def populate_testruns_for_upload_testruns(dbsession, bq: BQDriver):
-    testruns = []
+    testruns_written = [
+        MessageToDict(
+            ta_testrun_pb2.TestRun.FromString(testrun_bytes),
+            preserving_proto_field_name=True,
+        )
+        for testrun_bytes in mock_bigquery_service.mock_calls[0][1][3]
+    ]
+    assert snapshot("json") == sorted(testruns_written, key=lambda x: x["name"])
 
-    upload = UploadFactory()
-    upload.id_ = 1
-    dbsession.add(upload)
-    dbsession.flush()
 
-    testruns: list[test_results_parser.Testrun] = [
-        {  # this test is flaky failure
-            "name": "test_0",
-            "classname": "class_0",
-            "testsuite": "suite_upload",
-            "duration": 0.0,
-            "outcome": "failure",
-            "filename": None,
-            "computed_name": "upload_computed_name_0",
-            "failure_message": None,
-            "build_url": None,
-        },
-        {  # this test is just a failure
-            "name": "test_1",
-            "classname": "class_1",
-            "testsuite": "suite_upload",
-            "duration": 0.0,
-            "outcome": "failure",
-            "filename": None,
-            "computed_name": "upload_computed_name_1",
-            "failure_message": None,
-            "build_url": None,
-        },
-        {  # this test is a pass but also flaky
-            "name": "test_2",
-            "classname": "class_2",
-            "testsuite": "suite_upload",
-            "duration": 0.0,
-            "outcome": "pass",
-            "filename": None,
-            "computed_name": "upload_computed_name_2",
-            "failure_message": None,
-            "build_url": None,
-        },
-        {  # this test should be ignored
-            "name": "test_3",
-            "classname": "class_3",
-            "testsuite": "suite_upload",
-            "duration": 0.0,
-            "outcome": "pass",
-            "filename": None,
-            "computed_name": "upload_computed_name_3",
-            "failure_message": None,
-            "build_url": None,
-        },
+def test_pr_comment_agg(mock_bigquery_service, mock_config, snapshot):
+    driver = BQDriver(repo_id=1)
+    mock_bigquery_service.query.return_value = [
+        {
+            "commit_sha": "abc123",
+            "passed": 10,
+            "failed": 2,
+            "skipped": 1,
+            "flaky_failed": 1,
+        }
     ]
 
-    bq.write_testruns(None, 3, "abcde", "feature_branch", upload, "pytest", testruns)
+    result = driver.pr_comment_agg("abc123")
+
+    mock_bigquery_service.query.assert_called_once()
+    query, params = mock_bigquery_service.query.call_args[0]
+    assert snapshot("txt") == query
+    assert params == [
+        ScalarQueryParameter("repoid", "INT64", 1),
+        ScalarQueryParameter("commit_sha", "STRING", "abc123"),
+    ]
+    assert snapshot("json") == result
```

Comment:
here also, the snapshot just contains the `mock_bigquery_service.query.return_value`